core/test/integration/transactions/bitcoin_P2PKH_transaction_tests.cpp

```diff
@@ -226,4 +226,40 @@ TEST_F(ZCASHMakeP2SHTransaction, ParseSignedRawTransactionSapling) {
     auto tx = BitcoinLikeTransactionApi::parseRawSignedTransaction(currency, hex::toByteArray(strTx), networks::ZIP_SAPLING_PARAMETERS.blockHeight + 1);
     EXPECT_EQ(hex::toString(tx->serialize()), strTx);
 }
-*/
\ No newline at end of file
+*/
+
+TEST_F(BitcoinMakeP2PKHTransaction, UTXOPerformance) {
+    std::shared_ptr<AbstractWallet> w = wait(pool->createWallet("my_btc_wallet", "bitcoin_testnet", DynamicObject::newInstance()));
+    api::ExtendedKeyAccountCreationInfo info = wait(w->getNextExtendedKeyAccountCreationInfo());
+    info.extendedKeys.push_back("tpubDDDQZa67HazEpPFQWMh5QxK1F7QkAahVCXhBRzkQb2Qvb2jzZeoocaTeomYQb6JrA8962XPQFwainLPJsAZT9nTAsL8hewseSw7oFmi5Eq1");
+    info.derivations.push_back("44'/1'/1'");
+    std::shared_ptr<AbstractAccount> account = std::dynamic_pointer_cast<AbstractAccount>(wait(w->newAccountWithExtendedKeyInfo(info)));
+    std::shared_ptr<BitcoinLikeAccount> bla = std::dynamic_pointer_cast<BitcoinLikeAccount>(account);
+    Promise<Unit> p;
+    auto s = bla->synchronize();
+
+    s->subscribe(bla->getContext(), make_receiver([=](const std::shared_ptr<api::Event> &event) mutable {
+        fmt::print("Received event {}\n", api::to_string(event->getCode()));
+        if (event->getCode() == api::EventCode::SYNCHRONIZATION_STARTED)
+            return;
+        EXPECT_NE(event->getCode(), api::EventCode::SYNCHRONIZATION_FAILED);
+        EXPECT_EQ(event->getCode(),
+                  api::EventCode::SYNCHRONIZATION_SUCCEED_ON_PREVIOUSLY_EMPTY_ACCOUNT);
+        p.success(unit);
+    }));
+    Unit u = wait(p.getFuture());
```

Comment:
what is the purpose of the variable 'Unit u' ? I don't see any usage on this test